File: UnicoVehicle/UnicoVehicle.DTO/Accessories.cs
using System;
namespace UnicoVehicle.DTO
{
    public class Accessories
    {
        public int AccessoriesId { get; set; }
        public AccessoriesType AccessoriesType { get; set; }
        public string AccessoriesName { get; set; }
        public double Price { get; set; }
        public AccessoryBrand AccessoryBrand { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/Accessories.dart
import './AccessoriesBrand.dart';
import './AccessoriesType.dart';
import './VehicleName.dart';

class Accessories {
  final int? accessoriesId;
  final AccessoriesBrand? accessoriesBrand;
  final AccessoriesType? accessoriesType;
  final VehicleName? vehicleName;
  final String accessoriesName;
  final double price;

  Accessories({
    this.accessoriesBrand,
    this.accessoriesId,
    required this.accessoriesName,
    this.accessoriesType,
    required this.price,
    this.vehicleName,
  });
}
